### PR TITLE
Don't send bundle-only when re-offering, as described in

### PR DESCRIPTION
**Pull request by ekr**
BUNDLE S 6.

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2445,6 +2445,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>The "a=rtcp-rsize" line MUST only be added if present in
             the most recent answer.</t>
+
+            <t>An "a=bundle-only" line MUST NOT be added, as indicated
+            in <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation"/>,
+            Section 6. Instead, JSEP implementations simply omit ICE
+            parameters for m= lines which can only be used when bundled.
```
Comment (taylor-b) [inline draft-ietf-rtcweb-jsep.xml]: Should the second sentence be phrased as a MUST? Also, would "for m= sections" be more accurate than "for m= lines"?

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2445,6 +2445,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>The "a=rtcp-rsize" line MUST only be added if present in
             the most recent answer.</t>
+
+            <t>An "a=bundle-only" line MUST NOT be added, as indicated
+            in <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation"/>,
+            Section 6. Instead, JSEP implementations simply omit ICE
```
Comment (taylor-b) [inline draft-ietf-rtcweb-jsep.xml]: More generally, it's TRANSPORT and IDENTICAL category attributes that are omitted

**Comment by ekr** [comment]
I think we should hold this until Taylor's mmusic comments are resolved.

**Comment by ekr** [comment]
cf. https://mailarchive.ietf.org/arch/msg/mmusic/-jENmu4iaapy7NdJ9_SLHpNjSDc

**Comment by taylor-b** [comment]
It doesn't seem like BUNDLE is going to change at this point; the disadvantages of using "bundle-only" in subsequent offers are that non-BUNDLE-aware middleboxes may be unhappy with a port of "0", and also, the answerer can't reject the offerer BUNDLE-tag section without rejecting the whole group (https://github.com/cdh4u/draft-sdp-bundle/issues/22). I think the latter restriction is unnecessary (and it actually breaks some WebRTC max-bundle corner cases), but it doesn't sound like it's going to be changed.

Speaking of which, if we're being thorough, we should address those corner cases. In "creating an answer", we should specify that if the answer rejects all the offerered m= sections with nonzero ports, it must also reject the m= sections with "a=bundle-only". So, if you're offered transceivers A, B and C with "a=bundle-only", and you do `setRemoteDescription(sdp); A.stop();`, the answer will reject A _and_ B/C.

**Comment by fluffy** [comment]
talked to chairs and advice from chairs on this and similar things that depend on mmusic outcomes. We put in JSEP now what our best gueses is of how mmusic will end up and publish JSEP. At the same time we log a bug to check if we need to make any changes to line up with what mmusic decides. We can take changes to line up with mmusic as Last Call comments later on.

**Comment by ekr** [comment]
@juberti, @taylor-b: I think this should resolve it.

**Comment by juberti** [comment]
Fixes #509

**Comment by ekr** [comment]
@fluffy: this revised version needs your signoff + merge

**Pull request merged**